diff -ru gtk+-3.14.1.old/gtk/gtkapplication-quartz-menu.c gtk+-3.14.1/gtk/gtkapplication-quartz-menu.c
--- gtk+-3.14.1.old/gtk/gtkapplication-quartz-menu.c	2014-09-29 23:25:06.000000000 +0400
+++ gtk+-3.14.1/gtk/gtkapplication-quartz-menu.c	2014-09-30 12:37:14.722304833 +0400
@@ -30,6 +30,16 @@
 
 #import <Cocoa/Cocoa.h>
 
+#if !defined(MAC_OS_X_VERSION_10_7) || \
+MAC_OS_X_VERSION_MAX_ALLOWED < MAC_OS_X_VERSION_10_7
+
+@interface NSScreen (LionAPI)
+- (CGFloat)backingScaleFactor;
+- (NSRect)convertRectToBacking:(NSRect)aRect;
+@end
+
+#endif // 10.7
+
 #define ICON_SIZE 16
 
 #define BLACK               "#000000"
