diff --git a/gcr/Makefile.am b/gcr/Makefile.am
index 0834c67..59f32b7 100644
--- a/gcr/Makefile.am
+++ b/gcr/Makefile.am
@@ -161,12 +161,12 @@ gcr/gcr-dbus-generated.c: $(DBUS_XML_DEFINITIONS)
 	$(AM_V_GEN) sed -e 's/gcr_dbus/_gcr_dbus/g' \
 		gcr/gcr-dbus-generated.c > gcr/gcr-dbus-generated.c.tmp && \
 	mv gcr/gcr-dbus-generated.c.tmp gcr/gcr-dbus-generated.c
-
-gcr/gcr-dbus-generated.h: gcr/gcr-dbus-generated.c
 	$(AM_V_GEN) sed -e 's/gcr_dbus/_gcr_dbus/g' \
 		gcr/gcr-dbus-generated.h > gcr/gcr-dbus-generated.h.tmp && \
 	mv gcr/gcr-dbus-generated.h.tmp gcr/gcr-dbus-generated.h
 
+gcr/gcr-dbus-generated.h: gcr/gcr-dbus-generated.c
+
 pkgconfig_DATA += \
 	gcr-base-$(GCR_MAJOR).pc
 

